pkg/storage/tsdb/bucketindex/markers_bucket_client.go

```diff
@@ -100,9 +100,9 @@ func (b *globalMarkersBucket) IsObjNotFoundErr(err error) bool {
 	return b.parent.IsObjNotFoundErr(err)
 }
 
-// IsCustomerManagedKeyError returns true if the permissions for key used to encrypt the object was revoked.
-func (b *globalMarkersBucket) IsCustomerManagedKeyError(err error) bool {
-	return b.parent.IsCustomerManagedKeyError(err)
+// IsAccessDeniedErr returns true if the permissions for key used to encrypt the object was revoked.
```

Comment:
Do we need to update the comment?
`if the permissions for key used to encrypt the object was revoked` this seems specific to KMS error

Reply:
lemme do that.